src/components/bluebutton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const buttonStyle = {
```

Comment:
Styles should go in their own `.scss` file. Look at `Footer` and `HeaderInitial` for examples.